### PR TITLE
fix: make map works in two patterns with inline colon

### PR DESCRIPTION
**Pull request by Yiyiyimu**
Signed-off-by: yiyiyimu <wosoyoung@gmail.com>

### CODE REVIEW

**Comment by Yiyiyimu** [comment]
@spacewander PTAL

File: tinyyaml.lua [hunk 1]
```diff
@@ -510,7 +510,7 @@ local function parseseq(line, lines, indent)
     end
     local rest = ssub(line, j+1)
 
-    if sfind(rest, '^[^\'\"%s]*:%s') then
+    if sfind(rest, '^[^\'\"%s]*:$') or sfind(rest, '^[^\'\"%s]*:%s.') then
```
Comment (spacewander) [inline tinyyaml.lua:513]: Need to add comment for it

Comment (Yiyiyimu) [reply tinyyaml.lua:513]: Comments updated. PTAL~

Comment (spacewander) [reply tinyyaml.lua:513]: How about trailing whitespace after colon?

Comment (Yiyiyimu) [reply tinyyaml.lua]: @spacewander Thanks for pointing it out! Fixed

**Review by spacewander — APPROVED** [review]

**Pull request merged**